policy/templates/releng/.github/workflows/release.yml

```diff
@@ -7,7 +11,9 @@
 # - devenv ECR
 # - Cloudsmith
 
-name: Release
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
This will need to be quoted as it is a template. Something like `{{`${{ github.workflow }}-${{ github.ref }}`}}`